src/Core/Addon/Module/ModuleRepository.php

```diff
@@ -524,45 +525,9 @@ private function getModulesOnDisk($skip_main_class_attributes = false)
 
         return $modules;
     }
-    /*
-     * PROTECTED FUNCTIONS
-     */
-
-    /**
-     * In order to avoid class parsing, we generate a cache file which will keep mandatory data of modules.
-     *
-     * @param string $name The technical module name to find
-     *
-     * @return array Module data stored in file
-     */
-    private function generateCacheFile($data)
-    {
-        if (!$data) {
-            return;
-        }
-        $encoded_data = json_encode($data);
-        file_put_contents($this->cacheFilePath, $encoded_data);
-
-        return $data;
-    }
-
-    /**
-     * We load the file which contains cached data.
-     *
-     * @return array Module data loaded in file
-     */
-    private function readCacheFile()
-    {
-        // JSON file not found ? Generate it
-        if (!file_exists($this->cacheFilePath)) {
-            return array();
-        }
-        $data = json_decode(file_get_contents($this->cacheFilePath), true);
-
-        return ($data == null) ? array() : $data;
-    }
```

Comment:
These functions are not used anymore since we use Doctrine Cache, but they were brought back.